File: CursoJamiltonDamasceno/CursoWebCompleto/exer/estruturaSequencial/exer05.js
/*
5. Fazer um programa para ler o código de uma peça 1, o número de peças 1,
o valor unitário de cada peça 1, o código de uma peça 2,
o número de peças 2 e o valor unitário de cada peça 2.
Calcule e mostre o valor a ser pago.
 */

const code1 = 1234;
const qty1 = 20;
const price1 = 12.50;
const code2 = 4321;
const qty2 = 10;
const price2 = 35.50;

const totalValue = qty1 * price1 + qty2 * price2;

console.log(`Total da peça ${code1} e ${code2} é de R$${totalValue.toFixed(2)}`)

File: CursoLuizOtavio/CursoPython3/exer/estrutura_sequencial/exer05.py
"""
5- Fazer um programa para ler o código de uma peça 1, o número de peças 1,
o valor unitário de cada peça 1, o código de uma peça 2,
o número de peças 2 e o valor unitário de cada peça 2.
Calcule e mostre o valor a ser pago.
"""

cod_piece1 = int(input("Informe o código da peça 1: "))
num_piece1 = int(input("Informe a quantidade de peças 1: "))
price_piece1 = float(input("Informe o valor da peça 1: "))

cod_piece2 = int(input("Informe o código da peça 2: "))
num_piece2 = int(input("Informe a quantidade de peças 2: "))
price_piece2 = float(input("Informe o valor da peça 2: "))

value_piece1 = num_piece1 * price_piece1
value_piece2 = num_piece2 * price_piece2

print("O valor total de peças 1 é R$", value_piece1)
print("O valor total de peças 2 é R$", value_piece2)
